app/controllers/api/v1/posts_controller.rb

```diff
@@ -0,0 +1,47 @@
+module Api
+  module V1
+    class PostsController < ApiController
+      before_action :authorize_request
+
+      def list_posts
+        author_id = params[:user_id]
+        if author_id
+          posts = Post.where(author_id:)
+          render json: { status: 'SUCCESS', message: 'Loaded posts successfully', data: posts }, status: :ok
+        else
+          render json: { status: 'ERROR', message: 'User posts not found' }, status: :not_found
+        end
+      end
+
+      def list_comments
+        author_id = params[:user_id]
+        post_id = params[:post_id]
+        if author_id && post_id
+          comments = Comment.where(author_id:, post_id:)
```

Comment:
- [ ] You're missing the `jwt` gem which you used for your authentication.
- When added, make sure the endpoints are accessed with a user login. Meaning each API endpoint should be associated with a specific user

### View:
![image](https://github.com/gilberthappi/Blog-app/assets/50990925/366a51ea-c503-45ff-85c4-79d5fe7a520d)